config/database.php

```diff
@@ -38,7 +38,6 @@
 			'url' => env('DATABASE_URL'),
 			'database' => env('DB_DATABASE', database_path('database.sqlite')),
 			'prefix' => '',
-			'foreign_key_constraints' => env('DB_FOREIGN_KEYS', true),
```

Comment:
Does this definitely default to `true` if it's removed?

Reply:
No, it is used. See here: https://github.com/laravel/framework/blob/2bc180295ebf8c4848a6b9c4ae8fa0d00ec75217/src/Illuminate/Database/SQLiteConnection.php#L111-L114